reconcile/gitlab_labeler.py

```diff
@@ -10,6 +11,48 @@
 QONTRACT_INTEGRATION = 'gitlab-labeler'
 
 
+def guess_onboarding_status(changed_paths: List[str]) -> Optional[str]:
+    """
+    Guess the onboarding status of a given MR from the changed paths of the
+    MR.
+    """
+    apps = queries.get_apps()
+    #   logging.info(apps)
+    app_name = get_app_name(changed_paths)
+    if app_name is not None:
+        app = None
+        for a in apps:
+            if a['name'] == app_name:
+                app = a
```

Comment:
You want to `break` here. Ideally you'd mark the `name` field with `isSearchable: true` in the schema, and save yourself this linear search.

Reply:
what if the change set includes changes for several apps?

Reply:
Fair comment, then we should return `list[str]` instead of just `str`.